corehq/apps/translations/integrations/transifex/client.py

```diff
@@ -55,6 +55,15 @@ def get_resource_slugs(self, version):
             # get all slugs
             return [r['slug'] for r in all_resources]
 
+    def update_resource_slug(self, old_resource_slug, new_resource_slug):
```

Comment:
I'm a bit confused by this. Is it unused? Would it be needed to be called manually in a shell? Should there be some web endpoint or management command to use it?

Mostly asking, because I think if I saw this in code and looked around and saw it wasn't used, I'd probably end up deleting it.

Reply:
@emord 
was working on another PR where i needed this, 
https://github.com/dimagi/commcare-hq/pull/24652

Thought of adding it in a separate PR.
I added in the description that i would be using it, looks like it was not a good plan?

Reply:
Seems fine if you want to add it here. was just confused that it was not in the other PR since that was the only use